hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/utils/ActionStack.java

```diff
@@ -357,7 +357,7 @@ private ContractAction withUnsetRecipientIfNeeded(
         return (type == CONTRACT_CREATION) ? withUnsetRecipient(action) : action;
     }
 
-    // (FUTURE) Use builder for simplicity when PBJ lets us set the oneof recipient to UNSET;
+    // (FUTURE) Use builder for simplicity when PBJ lets us set the one of recipient to UNSET;
```

Comment:
don't do this one - `oneof` is actually a keyword in protobuf's specification DSL